go/tools/builders/nogo_fix.go

```diff
@@ -140,8 +155,9 @@ func getFixes(entries []diagnosticEntry, fileSet *token.FileSet) ([]fileChange,
 		}
 		if !foundApplicableFix {
 			allErrors = append(allErrors, fmt.Errorf(
-				"ignoring suggested fixes from analyzer %q at %s",
+				"ignoring suggested fixes from analyzer %q at %s. details about errors on each of the suggested fixes: %+v",
 				entry.analyzerName, fileSet.Position(entry.Pos),
+				perAnalyzerErrors,
```

Comment:
Printing the slice as it is is quite ugly, because it would become `[overlapping suggestions from "analyzer2" and "analyzer1" at xxx ]`. Let's print them line by line instead, or somehow format it properly.